real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/nodes/sensors/HillCrestImuNode.java

```diff
@@ -0,0 +1,110 @@
+package com.roboclub.robobuggy.nodes.sensors;
+
+
+import com.hcrest.jfreespace.Device;
+import com.hcrest.jfreespace.DeviceListenerInterface;
+import com.hcrest.jfreespace.DeviceStatistics;
+import com.hcrest.jfreespace.Discovery;
+import com.hcrest.jfreespace.DiscoveryListenerInterface;
+import com.hcrest.jfreespace.inreport.HidInMsg;
+import com.hcrest.jfreespace.outreport.FreespaceMsgOutDataModeControlV2Request;
+import com.hcrest.jfreespace.outreport.HidOutMsg;
+import com.roboclub.robobuggy.messages.HillCrestIMUMessage;
+import com.roboclub.robobuggy.ros.NodeChannel;
+import com.roboclub.robobuggy.ros.Publisher;
+
+import boofcv.alg.segmentation.ms.MergeSmallRegions.Node;
+
+public class HillCrestImuNode implements DiscoveryListenerInterface,DeviceListenerInterface, com.roboclub.robobuggy.ros.Node{
+	private Device device_;
+	private DeviceStatistics deviceStats;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'deviceStats' could be replaced by a local variable.](https://www.codacy.com/app/Mr-Trevor-Decker/RoboBuggy/file/2336478298/issues/source?bid=2938665&fileBranchId=3205314#l20)